src/caffe/proto/caffe.proto

```diff
@@ -181,6 +183,8 @@ message SolverParameter {
   optional int32 stepsize = 13;
   // the stepsize for learning rate policy "multistep"
   repeated int32 stepvalue = 34;
+  // the stepsize for learning rate policy "plateau"
+  repeated int32 plateau_winsize = 41;
```

Comment:
Thank you for your comment. I just updated the PR body.

Talking about `plateau_winsize`, I agree that it's not a bad idea to re-use `stepvalue` or `stepsize`. However, they specify _exact_ number of iterations between LR drops while `plateau_winsize` is more like a _minimum_ bound of of iterations between two consecutive LR drops. I thought sharing one parameter name might cause confusion.